### DISCUSSION TITLE
Any lighter version of bertopic?

### DISCUSSION THREAD

**Discussion by akarshghale** [MaartenGr/BERTopic#1488] [Q&A] [answered]
Hi,

I would like to know if there's a lighter version or a way to install and use bertopic without installing pytorch and other transformers models. I am not generating embeddings using this library so there's no need for me to use transformer models or pytorch. I am only concerned with the topic modelling only.

Your help is appreciated!

**Comment by MaartenGr** [comment, marked as answer]
There is! You can find more about it in the [documentation here](https://maartengr.github.io/BERTopic/getting_started/tips_and_tricks/tips_and_tricks.html#lightweight-installation).

**Reply by akarshghale** [reply]
Thanks!
